rootfs/api/serializers.py

```diff
@@ -474,3 +475,18 @@ class Meta:
         """Metadata options for a :class:`AppSettingsSerializer`."""
         model = models.AppSettings
         fields = '__all__'
+
+    def validate_whitelist(self, data):
```

Comment:
But it would also move us to use a different PG field from what I could gather.

Guess you want CIDR and IP to work with this?